Mathlib/CategoryTheory/Sites/Equivalence.lean

```diff
@@ -34,16 +34,62 @@ sufficiently small limits in the sheaf category on the essentially small site.
 
 -/
 
-universe u
+universe w v u
 
 namespace CategoryTheory
 
-open Functor Limits GrothendieckTopology
+open Functor Limits GrothendieckTopology Presheaf
 
 variable {C : Type*} [Category C] (J : GrothendieckTopology C)
 variable {D : Type*} [Category D] (K : GrothendieckTopology D) (e : C ≌ D)
 variable (A : Type*) [Category A]
 
+namespace Presheaf
+
+variable (H : C ⥤ D) [ConcreteCategory A] {F G : Dᵒᵖ ⥤ A} (f : F ⟶ G)
+
+lemma isLocallyInjective_whisker_isCocontinuous [H.IsCocontinuous J K] [IsLocallyInjective K f] :
```

Comment:
As `IsCocontinous` is the natural assumption, I do not think to include it in the name:
```suggestion
lemma isLocallyInjective_whisker [H.IsCocontinuous J K] [IsLocallyInjective K f] :
```